src/utils/getLanguageSlug.js

```diff
@@ -8,12 +8,13 @@ module.exports = (lang) => {
 
     /* Create array of all supported language codes with the country code trimmed. */
     const codes = config.languages.reduce((accumulator, currentValue) => {
-        accumulator.push(currentValue.code.split("-")[0]);
+        let lang = currentValue.code.split("-")[0];
+        accumulator[lang] = (++accumulator[lang] || 1);
```

Comment:
@amb26's suggestion to use `accumulator[lang] = (accumulator[lang] || 0) + 1;` would be clearer here. 

See: [comment](https://github.com/fluid-project/trivet/pull/148#discussion_r670799553).